pkg/features/features.go

```diff
@@ -58,8 +58,9 @@ const (
 
 	// SeedChange enables updating the `spec.seedName` field during shoot validation from a non-empty value
 	// in order to trigger shoot control plane migration.
-	// owner: @stoyanr
+	// owner: @stoyanr @plkokanov
```

Comment:
```suggestion
	// owner: @plkokanov
```

Reply:
Nit: we have also `@stoyanr` assigned to the `ForceRestrore` feature gate. Do we have someone that takes over the topic? If not, we can leave it as it is.

Reply:
Yep, I was planning to change it to myself as part of #6302

Reply:
Ok, so it will be tackled with other PR and not with this one, right?

Reply:
Okay, anyways, this can be changed later, no reason to block this PR because of it.